packages/components/src/internal/query/api.ts

```diff
@@ -160,8 +160,7 @@ export function applyQueryMetadata(rawQueryInfo: any, schemaName?: string, query
             views = views.asImmutable();
         }
 
-        // TODO get rid of the splitCamelCase?  It's only sometimes the right thing to do.
-        const queryLabel = Parsers.splitCamelCase(rawQueryInfo.title || _queryName);
+        const queryLabel = rawQueryInfo.title || _queryName;
```

Comment:
Hurray!

Reply:
Fall back to which `queryLabel`? Is there one coming from `rawQueryInfo`? We hadn't been using that before...

Reply:
sorry yes, I was talking about the prop on queryInfo. Something like this maybe?
`const queryLabel = rawQueryInfo?.title ?? rawQueryInfo?.queryLabel ?? _queryName;`

Though I'm not sure of what the case would be when a queryInfo has a queryLabel but not a title.